src/main/app/src/app/shared/edit/edit-datum-groep/edit-datum-groep.component.ts

```diff
@@ -106,7 +125,126 @@ export class EditDatumGroepComponent extends EditComponent {
         this.fataleDatumField.formControl.setValue(this.fataleDatum);
         this.showStreefError = false;
         this.showFataleError = false;
-
         this.editing = false;
     }
+
+    private maakOpschortDuurField(): InputFormField {
+        this.opschortDuurField = new InputFormFieldBuilder()
+        .id('duurDagen')
+        .label('opschortduur')
+        .validators(Validators.required, Validators.min(1))
+        .build();
+        return this.opschortDuurField;
+    }
+
+    private maakOpschortRedenField(): InputFormField {
+        this.opschortRedenField = new InputFormFieldBuilder()
```

Comment:
Zie ook vorig commentaar

Reply:
Klopt, maar heb ik al opgelost in het verdagen ticket.